### ISSUE TITLE
Milano Call for mentors - 24th November

### ISSUE THREAD

**Issue by piccoloaiutante** [nodeschool/italy#38]
Hi guys,

i've already started to organise a new nodeschool event in Milan on 26th of October. Who is up to mentor  at that event?

https://www.eventbrite.it/e/biglietti-nodeschool-28503072457

@teomurgi @simoneb

**Comment by teomurgi** [comment]
I’m available to mentor :)

**Comment by simoneb** [comment]
thanks @piccoloaiutante but I'm afraid that mid-week I can't make it up there

**Comment by diegobent** [comment]
sorry @piccoloaiutante but I can't

**Comment by piccoloaiutante** [comment]
thanks anyway guys @simoneb @diegobent

**Comment by teone** [comment]
@piccoloaiutante I'll be there!

**Comment by piccoloaiutante** [comment]
@teone that's awesome!

**Comment by lambrojos** [comment]
I will be there of course :)

**Comment by piccoloaiutante** [comment]
@lambrojos yeah you were counted by default ;-) BTW see you all saturday at NodeJS Conf IT

**Comment by piccoloaiutante** [comment]
due to the low attendees this node school will be postponed to 24 november. Who's still up?

**Comment by piccoloaiutante** [comment]
@lambrojos @simoneb @teone @diegobent @teomurgi nodeschool has been postponed to 24th november!

**Comment by simoneb** [comment]
@piccoloaiutante mid week I can't make it I'm afraid

**Comment by piccoloaiutante** [comment]
ping @teomurgi @lambrojos @teone @diegobent

**Comment by teomurgi** [comment]
@piccoloaiutante: sorry guys I'm busy that day

**Comment by lambrojos** [comment]
@piccoloaiutante I'm in
